### DISCUSSION TITLE
活動の要素だいぶ整理できた感ある

### DISCUSSION THREAD

**Discussion by motoi** [copilot-jp/project-sprint#375] [Development]
@KokoroKagawa @Satoshi-Matsumoto @yoneyama-tomohiro
https://github.com/copilot-jp/project-sprint/issues/343#issuecomment-1207296061
これだいぶ良い整理ができてきた気がする。ぜひみてまずディスカッションに無邪気なコメントください！
前提を説明してないけど米山さんもコメントほしい（どっかで説明したい！）

**Comment by KokoroKagawa** [comment]
好きポイント
- 成果と出力が双方向の矢印（⇔）でぎっちり紐づいていない。
タスクの発生にも出力の成果への貢献にも、良い意味であそびがある。
予測型のプロジェクトでも適応型のプロジェクトでもうまく当てはめられそう。
- 成果自体の変容について描かれている
出力が**成果の変容**に貢献することは、これまでの一枚絵では描けていなかったし、orgの記述においても（「タスクの実行がテンションやアイデアを生む」という意識はありつつも）明確に書かれてこなかった。でもとても大事なポイントだと思う。
プロジェクトゴールが決められない・マイルストーンが引けない問題もこれで解決に向かいそう。
